src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_enkf_update.F90

```diff
@@ -1367,6 +1374,46 @@ subroutine apply_enkf_increments( N_catd, N_ens, update_type, &
        end do
 
        cat_progn_has_changed = .true.
+     
+     case(11) select_update_type ! snow update !jpark50
+           if (logit) write (logunit,*) 'applying Snow data increments'
+
+       if ( date_time%hour /= 0 .and. &
+            date_time%min  /= 0 .and. &
+            date_time%sec  /= 0         ) then
+
+          if (logit) write (logunit,*) 'no application of increments at this time'
+          cat_progn_has_changed = .false.
+          return
+
+       end if
+
+       cat_progn_has_changed = .true.     ! conservative initialization
```

Comment:
The statement: `cat_progn_has_changed = .true. ! conservative initialization` does not make sense here.  This initialization is already at the beginning of the subroutine.